hack/check-files-alignment.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash -e
+
+#
+# Copyright 2019 IBM Corp.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+source hack/project_info.sh
+roles_yaml_path=config/rbac/role.yaml
+origin_crd_yaml_path=config/crd/bases/csi.ibm.com_ibmblockcsis.yaml
```

Comment:
```suggestion
roles_yaml_path=config/rbac/role.yaml
crd_yaml_path=config/crd/bases/csi.ibm.com_ibmblockcsis.yaml
```
(similar to why `roles_yaml_path` shouldn't be `origin_roles_yaml_path`)

Reply:
I added the origin because there are other crd YAML files and there isn't any other role YAML file, so this is why I added the origin.
Do you still want me to change it? or keep it?

Reply:
I think "origin" can be useful only in a context dealing with destination (usually a multidoc yaml, e.g. a csv yaml).
so please change it here, but feel free to use "origin" in later code where there is destination as well, e.g.
```
verify_yaml_doc_in_multidoc_yaml (){
origin_crd_yaml_path = $1
destination_yaml_path = $2
.
.
}
verify_yaml_doc_in_multidoc_yaml crd_yaml_path operator_yaml_path
```
though IMO when it comes to the diff verifications in this file, they are more of an "inclusion" (/"containment") check.
I think source and destination are more suitable where an actual copy operation is performed, because there is more focus on the direction

Reply:
done, changed the `origin_crd_yaml_path` variable

Reply:
I meant that we are missing a `verify_yaml_doc_in_multidoc_yaml` function that receives parameters

Reply:
@oriyarde I agree that a more generic method is generally better.